app/src/main/java/org/sopt/and/SignInActivity.kt

```diff
@@ -0,0 +1,202 @@
+package org.sopt.and
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.SnackbarHost
+import androidx.compose.material3.SnackbarHostState
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import org.sopt.and.User.EMAIL
+import org.sopt.and.User.PASSWORD
+import org.sopt.and.component.EmailTextField
+import org.sopt.and.component.LeftButtonTopBar
+import org.sopt.and.component.PasswordTextField
+import org.sopt.and.ui.theme.ANDANDROIDTheme
+
+class SignInActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        val intent = intent
+        val email = intent.getStringExtra(EMAIL) ?: ""
+        val password = intent.getStringExtra(PASSWORD) ?: ""
+
+        setContent {
+            ANDANDROIDTheme {
+                Scaffold(modifier = Modifier.fillMaxSize()) { innerPadding ->
+                    SignInScreen(
+                        modifier = Modifier.padding(innerPadding),
+                        email = email,
+                        password = password,
+                        onSignInSuccess = { userEmail ->
+                            navigateToMyPageScreen(userEmail)
+                        },
+                        navigateToSignUpScreen = {
+                            navigateToSignUpScreen()
+                        }
```

Comment:
람다를 사용한다면 내부적 remember가 있지만 람다 자체가 재할당 되는 경우가 생겨요.
리컴포지션에서의 성능을 챙기려면 람다 없이 사용할 수 있는 코드는 그냥 사용하는건 어떨까요?

```suggestion
                        onSignInSuccess = navigateToMyPageScreen,
                        navigateToSignUpScreen = navigateToSignUpScreen
```

Reply:
어어ㅓ어ㅓ으어... 왜 잘 안될까요... 조금 더 싸워보고 연락드려보겠습니다!